apis/v1alpha1/route_types.go

```diff
@@ -140,6 +142,7 @@ type RouteGatewayStatus struct {
 	// route has been admitted or rejected by the Gateway, and why.  Note
 	// that the route's availability is also subject to the Gateway's own
 	// status conditions and listener status.
+	// +kubebuilder:validation:MaxItems=8
```

Comment:
Same as https://github.com/kubernetes-sigs/service-apis/pull/336/files?file-filters%5B%5D=.go&file-filters%5B%5D=.html&file-filters%5B%5D=.md&file-filters%5B%5D=.yaml&hide-deleted-files=true#r497816693

Reply:
I think this should also be a required field because `RouteGatewayStatus` would only exist if there were a condition to set.